test/functional/ui/messages2_spec.lua

```diff
@@ -418,4 +419,23 @@ describe('messages2', function()
                                                            |
     ]])
   end)
+
+  it('FileType is fired after default options are set', function()
+    n.exec([[
+      let g:optionset = 0
+      au FileType pager set nowrap
```

Comment:
this does not have `++nested` so the `set nowrap` itself will not trigger OptionSet, right? a bit confusing given the test context.